### ISSUE TITLE
questions on codes

### ISSUE THREAD

**Issue by JiaQiSJTU** [MaoGWLeon/SMN_Pytorch#3]
candidate_scores = F.softmax(candidate_scores, 0).cpu().detach().numpy()

why the axis for softmax is 0 ? Shouldn't it be -1 (or 1)?
